### DISCUSSION TITLE
Syntax Confusion

### DISCUSSION THREAD

**Discussion by Palak-K9** [neo4j-graphql/neo4j-graphql-java#315] [Q&A] [answered]
Hi,

I am following the below example, and using neo4j-graphql-java1.8.0 version.
[Given example](https://github.com/neo4j-graphql/neo4j-graphql-java/blob/master/core/src/test/resources/issues/gh-112.adoc)

<img width="530" alt="ss3" src="https://github.com/neo4j-graphql/neo4j-graphql-java/assets/79653860/009fb066-7cde-42de-b52d-2616c5444abb">

but I am not able to write the query without "**where**" clause for uuid. But I want to write the queries as shown in the example. Am I missing something?

**Comment by Andy2003** [comment]
You need to provide more information:

- what ist the error you are getting
- what is the schema (minimal example)
- how does the request look like

You referenced the example: Maybe you just need to remove the argument `uuid`?

**Reply by Palak-K9** [reply, marked as answer]
I wanted to query users like the following - 
```
query user( $uuid: ID ){
  user(uuid: $uuid) {
    uuid
    name
    associates { name uuid }
  }
}
```

But, following the above query wasn't giving me the correct result i.e. it was giving me users but not with the uuid, I was passing.

So, I had to query using the "where" keyword. The below query is working fine for me and its result is what I want to achieve without using the **where** keyword.
```
query user( $uuid: ID ){
  user(where: {uuid: $uuid}) {
    uuid
    name
    associates { name uuid }
  }
}
```

I found out that I have to set "useWhereFilter" to **false** in the GraphQLConfiguration.java class.

```
{
  "queryOptionStyle": "INPUT_TYPE",
  "useWhereFilter": false
}
```

Now, I am able to get the filtered users without using the **where** keyword. My query is resolved
